packages/contractkit/src/utils/ber-utils.ts

```diff
@@ -0,0 +1,33 @@
+import { ensureLeading0x } from '@celo/utils/lib/address'
+import * as asn1 from 'asn1js'
+
+export const toArrayBuffer = (buffer: Buffer): ArrayBuffer => {
```

Comment:
It looks like we can just use the built in [.buffer property](https://nodejs.org/docs/latest/api/buffer.html#buffer_buf_buffer) in node versions >= 4.x
https://stackoverflow.com/a/12101012/677846